src/platform_impl/web/event_loop/runner.rs

```diff
@@ -234,64 +234,74 @@ impl Shared {
-                    debug_assert_eq!(
-                        pointer_type, "mouse",
-                        "expect pointer type of a chorded button event to be a mouse"
-                    );
+                    // chorded button event
+                    let device_id = RootDeviceId(DeviceId(event.pointer_id()));
 
-                    let state = if backend::event::mouse_buttons(&event).contains(button.into()) {
-                        ElementState::Pressed
-                    } else {
-                        ElementState::Released
-                    };
+                    if let Some(button) = backend::event::mouse_button(&event) {
+                        debug_assert_eq!(
+                            pointer_type, "mouse",
+                            "expect pointer type of a chorded button event to be a mouse"
+                        );
 
-                    runner.send_event(Event::DeviceEvent {
-                        device_id,
-                        event: DeviceEvent::Button { button: button.to_id(), state },
-                    });
+                        let state = if backend::event::mouse_buttons(&event).contains(button.into())
+                        {
+                            ElementState::Pressed
+                        } else {
+                            ElementState::Released
+                        };
 
-                    return;
-                }
+                        runner.send_event(Event::DeviceEvent {
+                            device_id,
+                            event: DeviceEvent::Button { button: button.to_id(), state },
+                        });
 
-                // pointer move event
-                let mut delta = backend::event::MouseDelta::init(&window, &event);
-                runner.send_events(backend::event::pointer_move_event(event).flat_map(|event| {
-                    let delta = delta.delta(&event).to_physical(backend::scale_factor(&window));
+                        return;
+                    }
 
-                    let x_motion = (delta.x != 0.0).then_some(Event::DeviceEvent {
-                        device_id,
-                        event: DeviceEvent::Motion { axis: 0, value: delta.x },
-                    });
+                    // pointer move event
+                    runner.send_events(backend::event::pointer_move_event(event).flat_map(
+                        |event| {
+                            let delta = delta.delta(&event);
 
-                    let y_motion = (delta.y != 0.0).then_some(Event::DeviceEvent {
-                        device_id,
-                        event: DeviceEvent::Motion { axis: 1, value: delta.y },
-                    });
+                            if delta.x == 0 && delta.y == 0 {
+                                return None.into_iter().chain(None).chain(None);
+                            }
 
-                    x_motion.into_iter().chain(y_motion).chain(iter::once(Event::DeviceEvent {
```

Comment:
Why get rid of `iter::once`?

Reply:
Because the type has to match with [this line](https://github.com/daxpedda/winit/blob/bbf16bc6ee66320ab1353c2cf46b9a6e922f107b/src/platform_impl/web/event_loop/runner.rs#L280).
So I had to change it to `Option` to use `None` above so I can deliver an empty iterator.